driver-core/src/test/functional/com/mongodb/internal/operation/AggregateToCollectionOperationSpecification.groovy

```diff
@@ -331,27 +330,6 @@ class AggregateToCollectionOperationSpecification extends OperationFunctionalSpe
         async << [true, false]
     }
 
-    @IgnoreIf({ !serverVersionAtLeast(3, 6) || (serverVersionAtLeast(4, 1) && isSharded()) })
```

Comment:
This test depends on being able to explain this operation.  Since I removed that capability, I just removed this test.  It seems like this is unlikely to break, but if it's of concern I can figure out a different way to test this

Reply:
Ack